__tests__/app.test.js

```diff
@@ -32,4 +32,26 @@ describe("App", () => {
         });
     });
   });
+  describe("GET /api/reviews", () => {
+    test(`200: Responds with and array of review objects, each of which should have an owner, title, review_id, category, review_img_url, created_at, votes, designer, and comment_count property`, () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body, body: { reviews } }) => {
+          expect(body).toHaveProperty("reviews");
+          expect(reviews.length).toBeGreaterThanOrEqual(1);
+          reviews.forEach((review) => {
+            expect(review).toHaveProperty("owner", expect.any(String));
+            expect(review).toHaveProperty("title", expect.any(String));
+            expect(review).toHaveProperty("review_id", expect.any(Number));
+            expect(review).toHaveProperty("category", expect.any(String));
+            expect(review).toHaveProperty("review_img_url", expect.any(String));
+            expect(review).toHaveProperty("created_at", expect.any(String));
+            expect(review).toHaveProperty("votes", expect.any(Number));
+            expect(review).toHaveProperty("designer", expect.any(String));
+            expect(review).toHaveProperty("comment_count", expect.any(Number));
+          });
+        });
+    });
+  });
```

Comment:
We need a test to prove that these are sorted in descending order. Try taking a look at the docs for `jest-sorted`.

Reply:
Knew there was something I was missing